src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Configuration;
+using Microsoft.Azure.NotificationHubs;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
+{
+    /// <summary>
+    /// Defines the configuration options for the NotificationHubs binding.
+    /// </summary>
+    public class NotificationHubsConfiguration : IExtensionConfigProvider
+    {
+        internal const string NotificationHubConnectionStringName = "AzureWebJobsNotificationHubConnectionString";
```

Comment:
Since the product is plural and you use that elsewhere, wondering if this shouldn't be AzureWebJobsNotificationHubsConnectionString?